File: templates/Skill-Template/src/csharp/SkillTemplate/Bot Framework/SkillTemplate/SkillConversationState.cs
using Microsoft.Bot.Builder.Dialogs;

namespace $safeprojectname$
{
    public class SkillConversationState : DialogState
    {
        public SkillConversationState()
        {
        }

        public string Token { get; internal set; }

        public void Clear()
        {
        }
    }
}

File: templates/Skill-Template/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/src/_skillConversationState.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License

import { DialogState } from 'botbuilder-dialogs';

/**
 * Here is the documentation of the SkillConversationState class
 */
export interface ISkillConversationState extends DialogState {
    token?: string;
    // tslint:disable-next-line:no-any
    luisResult?: any;

    clear(): void;
}
